src/System.Private.ServiceModel/tests/Common/Infrastructure/Infrastructure.Common.csproj

```diff
@@ -16,6 +16,7 @@
     <ProjectGuid>{AFD69665-89A3-42AE-A32E-AB2CBBE6EE7E}</ProjectGuid>
     <CLSCompliant>false</CLSCompliant>
     <IsTestProject>true</IsTestProject>
+    <NugetTargetMoniker Condition="'$(NugetTargetMoniker)' ==''">.NETStandard,Version=v1.3</NugetTargetMoniker>
```

Comment:
I think this is redundant now that you default the targetgroup.  Setting targetgroup should trigger https://github.com/dotnet/wcf/blob/master/dir.props#L343 which sets NuGetTargetMoniker.